Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cli"
-version = "0.2.0"
+version = "0.2.1"
```

Comment:
Why ?

Reply:
Because the code has changed. We're using versions to tag a specific state of software. If that state changes, the version has to change.

Reply:
No. We bump version when we release new version, not for every fix (this is not even enough as the version is referenced in docker and somewhere else)

Reply:
How about using dash notation for intermediary versions under development? Like `0.2.1-1` and while releasing we can get rid of the dash suffix.

Reply:
I don't see any benefit to that as you can't download it from any place. And it's what we do in all our projects so what's the point of doing differently here?

Reply:
We're testing it here and there and we should be providing it to other teams for tests before releasing it. And every software iteration has to carry a different tag. Reports have to include that version info to match the exact state of that software. You can use build number, commit hash, or anything else. But some info has to indicate that state. Version is the easiest one.

Reply:
When it's unreleased that's what you have git commit hash for. We are *not* providing that as a binary for testing to anyone.

Reply:
Guys. We use semantic versioning. We modify the version when we release. 
If you for whatever reason want to modify the version between releases you can use -rc tail.

It mains release candidate. 

2.0.0-rc1. 
2.0.0-rc2.
2.0.0-rc3.

We also can release the same version multiple times. We had to do it recently, to trigger the upgrade on UC16 devices. Then you can add -r tail

It means - revision.

2.0.0-r1 means 2.0.0 revision 1.